app/lib/actions/katello/organization/manifest_refresh.rb

```diff
@@ -6,7 +6,7 @@ class ManifestRefresh < Actions::AbstractAsyncTask
 
         include Helpers::Notifications
 
-        def plan(organization)
+        def plan(organization) # rubocop:disable Metrics/MethodLength
```

Comment:
Maybe you can move the code block which does the repo sync stuff into a separate method to avoid disabling this?

```
do_repo_syncs_and_stuff if manifest_update
```